### ISSUE TITLE
Tooltip is not working in charts

### ISSUE THREAD

**Issue by devpkr1** [dbcodeio/public#30]
Hi mike ,

I have encountered with new issue while creating charts, i have found that tooltip is not working.
I am attaching the screen recording 

https://github.com/user-attachments/assets/6080351e-c0a6-43e1-9a0d-095215e0ba6c

Best Regards,
Pradeep Kumar

**Comment by mikeburgh** [comment]
Yeah, not sure why.. I cannot get it to work in any configuration.. It seems like its not getting any mouse co-ords... need to dig into further

**Comment by devpkr1** [comment]
Hi Mike,

I've noticed your latest update, and the tooltip feature is working; however, there’s a minor issue. The tooltip doesn’t appear on the charts until we change the chart type at least once.


Best regards,
Pradeep Kumar

**Comment by devpkr1** [comment]
Hi Mike,

I've noticed your latest update, and the tooltip feature is working; however, there’s a minor issue. The tooltip doesn’t appear on the charts until we change the chart type at least once.

Screen Recording attached
https://github.com/user-attachments/assets/eff59596-3d3b-41d8-9956-997ad66f9cf9


Best regards,
Pradeep Kumar

**Comment by mikeburgh** [comment]
Thanks, I also noticed the chart goes away if you right click after it first loads.. still looking into what causes it, and if I can replicate it out side VScode I will raise it upstream

**Comment by devpkr1** [comment]
Got it, thanks for looking into it!

**Comment by devpkr1** [comment]
Hi Mike,

I've noticed that whenever we right-click on the chart, not only does the chart disappear, but the entire output result vanishes as well. Please see the attached screenshot.

![Image](https://github.com/user-attachments/assets/3edae5d9-d1d3-463c-b15a-ccb774edfa7b)

Thanks,
Pradeep

**Comment by mikeburgh** [comment]
Fixed, will update when released.

**Comment by mikeburgh** [comment]
Released in 1.7.3
